### PR TITLE
minio-mc 20160821030249

### PR DESCRIPTION
**Pull request by nl5887**
- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [X] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/minio-mc.rb [hunk 1]
```diff
@@ -24,8 +24,18 @@ def install
     clipath.install Dir["*"]
 
     cd clipath do
-      system "go", "build", "-o", bin/"mc"
+      if build.head?
+        system "go", "build", "-o", buildpath/"mc"
```
Comment (DomT4) [inline Formula/minio-mc.rb]: Why the change here to the simpler mechanism becoming a head-only thing? Did something break in current stable?

Comment (nl5887) [inline Formula/minio-mc.rb]: We want the tagged release to show the correct version information in `mc version`. The head version will show DEVELOPMENT_GOGET.

**Comment by apjanke** [comment]
Merged.

Thank you for your contribution to Homebrew!
